scripts/scripts.js

```diff
@@ -30,6 +30,17 @@ const LCP_BLOCKS = [...Object.values(ARTICLE_TEMPLATES), 'hero']; // add your LC
 
 const range = document.createRange();
 
+// getting real path, and adjusting canonical link to use the vanity path
+const canonicalLinkTag = document.head.querySelector('link[rel="canonical"]');
+if (canonicalLinkTag) {
+  const longPathMetadata = document.createElement('meta');
+  longPathMetadata.setAttribute('property', 'hlx:long-form-path');
```

Comment:
@DriftingSands this property will exist and be named "path" once @hsaginor reimports the whole content